releases/release-1.10/release-notes-draft.md

```diff
@@ -50,6 +50,8 @@ In terms of networking, Kubernetes 1.10 is about control. Users now have beta su
 
 # Before Upgrading
 
+* If you need to downgrade from 1.10 to 1.9.x, downgrade to v1.9.6 to ensure PV and PVC objects can be deleted properly.
```

Comment:
Do we support multi-release downgrades?  ie, could the user downgrade from 1.10.x to 1.8.x?